extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -341,6 +313,106 @@ void openTracingIntegration(Capabilities capabilities,
         return classes;
     }
 
+    // Services Integrations
+
+    @BuildStep
+    void activateMetrics(Capabilities capabilities,
+            Optional<MetricsCapabilityBuildItem> metricsCapability,
+            BuildProducer<SystemPropertyBuildItem> systemProperties,
+            BuildProducer<UnremovableBeanBuildItem> unremovableBeans) {
+
+        boolean activate = shouldActivateService(capabilities, quarkusConfig.metricsEnabled, metricsCapability.isPresent(),
+                "metrics", "quarkus.smallrye-graphql.metrics.enabled");
+        if (activate) {
+            if (metricsCapability.isPresent() && metricsCapability.get().metricsSupported(MetricsFactory.MP_METRICS)) {
```

Comment:
Isn't Micrometer the extension that should be used for Metrics now? Not sure if it's supported by your GraphQL extension though?

/cc @ebullient @jmartisk

Reply:
there is an additional issue open to sort out graphql and metrics.. rgis will be revisitedvat that time.

Reply:
I am leaving the Metrics as is, and we can do a new PR (@jmartisk can help) to update to the micrometer.